x-pack/plugins/observability_solution/observability/server/assets/component_templates/slo_mappings_template.ts

```diff
@@ -45,6 +45,24 @@ export const getSLOMappingsTemplate = (name: string): ClusterPutComponentTemplat
             },
           },
         },
+        // Synthetics specific fields
+        monitor: {
```

Comment:
@shahzad31 Do you think I should map `monitor.name` and `observer.geo.name` too?

Reply:
yeah i think it makes sense